src/main/resources/templates/list-jobs.html

```diff
@@ -12,6 +12,37 @@ <h1 th:text="${#strings.capitalizeWords(title)}"></h1>
 
     <!-- TODO #1 - Use a loop to display job results in a table. -->
 
+    <table class="job-listing" th:each="job : ${jobs}">
+        <tr>
+            <td>ID</td>
+            <td th:text="${job.getId()}"></td>
```

Comment:
This works but you generally want to access the field directly using `job.id` rather than `job.getId()`. There's a lot of overhead when it comes to calling the `getId()` method each time, so `job.id` is preferred. Although it does work!